app/src/main/res/layout/layout_keycard_intro.xml

```diff
@@ -4,7 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:background="@color/pale_grey_two">
+    android:background="@color/pale_grey_two">
```

Comment:
Isn't this already set in the AndroidManifest? Do avoid overdraw we should not set it here

Reply:
ok, using @style/GreyBackgroundTheme then